packages/babel-traverse/src/scope/index.js

```diff
@@ -830,7 +832,7 @@ export default class Scope {
         return scope;
       }
     } while ((scope = scope.parent));
-    throw new Error("We couldn't find a Function or Program...");
+    return null;
```

Comment:
because in another instance of the code, it freaks out https://github.com/babel/babel/pull/5923#discussion_r127225960

maybe there's a better way to deal with it though, open to suggestions :)